From 5952beca73f6372d73ad437bfb53fbdbbdcf53a7 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Fri, 20 Jun 2025 22:55:01 +0800
Subject: [PATCH 8/8] AOSCOS: gsgpu: remove VBIOS check requirement for
 gsgpu_init()

On Lemote B2219, the VBIOS is for some reason considered invalid by the
driver (but it does work on UOS 20 <= 1070).

Nuke this check as a test.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 gsgpu/gsgpu_drv.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/gsgpu/gsgpu_drv.c b/gsgpu/gsgpu_drv.c
index 94d9f88..33de0c6 100644
--- a/gsgpu/gsgpu_drv.c
+++ b/gsgpu/gsgpu_drv.c
@@ -625,11 +625,6 @@ static int __init gsgpu_init(void)
 		filp_close(fw_file, NULL);
 	}
 
-	if (!check_vbios_info()) {
-		DRM_INFO("gsgpu can not support this board!!!\n");
-		return -EINVAL;
-	}
-
 	r = gsgpu_sync_init();
 	if (r)
 		goto error_sync;
-- 
2.49.0

